source/includes/_elixir.md

```diff
@@ -383,6 +384,54 @@ defmodule FirestormWeb.Web.PostsChannel do
   end
 ```
 
+### Plug Chunked Response (HTTP Streaming)
+
+In a Plug application, a chunked response needs to be instrumented directly, rather than relying on
+the default Scout instrumentation Plug. The key part is to `start_layer` beforehand, and then call
+`before_send` after the chunked response is complete.
+
+```
+def chunked(conn, _params) do
+  # The "Controller" argument is required, and should not be changed. The second argument is the
+  # name this endpoint will appear as in the Scout UI. The `action_name` function determines this
+  # automatically.
+  ScoutApm.TrackedRequest.start_layer("Controller", ScoutApm.Plugs.ControllerTimer.action_name(conn))
+
+  conn =
+    conn
+    |> put_resp_content_type("text/plain")
+    |> send_chunked(200)
+
+  {:ok, conn} =
+    Repo.transaction(fn ->
+      Example.build_chunked_query(...)
+      |> Enum.reduce_while(conn, fn data, conn ->
+        case chunk(conn, data) do
+          {:ok, conn} ->
+            {:cont, conn}
+
+          {:error, :closed} ->
+            {:halt, conn}
+        end
+      end)
+    end)
+
+  ScoutApm.Plugs.ControllerTimer.before_send(conn)
+
+  conn
+end
+```
+
+Then have the default instrumentation ignore the endpoint (since it is manually instrumented now):
+
+```
+config :scout_apm,
+  name: "My Scout App Name",
+  key: "My Scout Key",
+  ignore: ["/chunked"]
```

Comment:
So this ignored name should be the name of `action_name`?

Reply:
it should be the url prefix of the action. I'll mention that.